pkg/common/config/config.go

```diff
@@ -455,14 +459,15 @@ func GetGCconfig(ctx context.Context, cfgPath string) (*Config, error) {
 			return cfg, err
 		}
 	}
-	// Set default GCPort if Port is still empty
+	// Set default GCPort if Port is still empty.
 	if cfg.GC.Port == "" {
 		cfg.GC.Port = DefaultGCPort
 	}
 	return cfg, nil
 }
 
-// validateGCConfig validates the Guest Cluster config contains all the necessary fields
+// validateGCConfig validates the Guest Cluster config contains all the
```

Comment:
validates the -> validates that

Reply:
Do you mind if I use a different change for this preexisting issue?

Reply:
Sure. Since you were fixing the comments, I thought it would be good to fix this as well.